src/BaseAPI.php

```diff
@@ -1693,12 +1694,58 @@ public function deleteEdgeRule(
      * @return BunnyClientResponseInterface
      * @param int $pullZoneId
      * @param array<string,mixed> $body
+     *
+     * @deprecated since 4.5 (to be removed in 5.0). Use {@link addEdgeRule} or {@link updateEdgeRule} instead.
```

Comment:
Note:

This is backwards-compatible and therefore will be released in `4.5.0`.